packages/badge/src/css/index.js

```diff
@@ -1,10 +1,10 @@
-import { transparentize } from '@pluralsight/ps-design-system-util/color'
+import { transparentize } from '@pluralsight/ps-design-system-util/color.js'
```

Comment:
not sure why but the extension seems weird on imports...thoughts?

Reply:
Yes, it does seem weird.  I have missed it in my conceptualizing of what is a file import vs a named import.  But the lint rule you added somewhere at some point caught it.  

I am all for adding it to move away from magic module resolution and support the new browser standards.